src/test/java/uk/gov/hmcts/reform/sendletter/logging/AppInsightsTest.java

```diff
@@ -109,6 +120,36 @@ public void should_track_message_acknowledgement_event_for_fail_case() {
         verifyNoMoreInteractions(telemetry);
     }
 
+    @Test
+    public void should_track_event_of_not_printed_letter() {
+        StaleLetter letter = mock(StaleLetter.class);
```

Comment:
May be you could create an actual object here rather than mock and then in below verification also assert the values?

Reply:
remnant of the past. will update test